### PR TITLE
Enable DEBUG=1 when using bundler related task

### PR DESCRIPTION
**Pull request by ph**
When running any logstash's command you can use DEBUG=1 which should
give the user a bit more information about what LS is doing.

This PR changes the behavior of bundler to make sure its run with the
`--verbose` flag.

### CODE REVIEW

File: lib/bootstrap/bundler.rb [hunk 1]
```diff
@@ -104,13 +104,22 @@ def invoke!(options = {})
       ::Bundler.settings[:gemfile] = LogStash::Environment::GEMFILE_PATH
       ::Bundler.settings[:without] = options[:without].join(":")
 
+      if !debug?
+        # Deal with transient network errors
+        return execute_bundler_with_retry(options)
```
Comment (jsvd) [inline lib/bootstrap/bundler.rb]: any reason for the explicit return?

**Comment by jsvd** [comment]
LGTM

File: lib/bootstrap/bundler.rb (hunk 1 above)
Comment (ph) [inline lib/bootstrap/bundler.rb]: Right there is no need for that.

**Comment by ph** [comment]
@jsvd which branch should we target this?

**Comment by jsvd** [comment]
All...master 5.0 2.x 2.3

**Pull request closed without merging**